frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -218,12 +228,23 @@
     }
 
     function exportExcel(){
+        $('#keywordsearchinput').val("");
```

Comment:
I do not think that this is necessary. You leave the page immediately at the end of `exportExcel()`